LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Call-Em-All
+Copyright (c) 2014-2016 Call-Em-All
```

Comment:
These have to be updated every year. and the copyright holder "owns" the product for 50-60 years (if I remember right). This product is likely not registered anyway. so this copyright notice is more like common-sense than law-suit material. I wouldn't bother with it, after all it MIT license anyway :grin: But if anyone should edit this notice it's board members (or founders) of call-em-all not any of us. since we don't own this project.